### PR TITLE
Fill out README with feed action details

### PR DESCRIPTION
**Pull request by bjustin-ibm**
This is by no means a "comprehensive" doc. For example, there is currently nothing about building and deploying this service yourself. That will come later.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,95 @@
...
+|kafka_brokers_sasl|JSON Array of Strings|This parameter is an array of `<host>:<port>` strings which comprise the brokers in your Message Hub instance|
+|user|String|Your Message Hub user name|
+|password|String|Your Message Hub password|
+|topic|String|The topic you would like the trigger to listen to|
+|kafka_admin_url|URL String|The URL of the Message Hub admin REST interface|
+|api_key|String|Your Message Hub API key|
+|isJSONData|Boolean (Optional - default=false)|When set to `true` this will cause the feed to attempt the message content as JSON before passing it along as the trigger payload.|
+
+While this list of parameters may seem daunting, they can be automatically set for you by using the `package refresh` CLI command.
+
```
Comment (dubeejw) [inline README.md]: Give me an example to create a trigger using the Message Hub feed like you did for Kafka.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,95 @@
...
+After creating a trigger, the system will monitor the specified topic in your messaging service. When new messages are posted, the trigger will be fired.
+
+The payload of that trigger will contain a `messages` field which is an array of messages that have been posted since the last time your trigger fired. Each message object in the array will contain the following fields:
+- topic
+- partition
+- offset
+- key
+- value
+
+In Kafka terms, these fields should be self-evident. However, the `value` requires special consideration. If the `isJSONData` parameter was set `false` (or not set at all) when the trigger was created, the `value` field will be the raw value of the posted message. However , if `isJSONData` was set to `true` when the trigger was created, the system will make attempt to parse this value as a JSON object, on a best-effort basis. If parsing is successful, then the `value` in the trigger payload will be the resulting JSON object.
```
Comment (dubeejw) [inline README.md]: Remove between "However" and ",".

File: README.md [hunk 3]
```diff
@@ -1,4 +1,95 @@
...
+
+For example, if I post a message of `{"title": "Some string", "amount": 5, "isAwesome": true}` with `isJSONData` set to `true`, my trigger payload might look something like this:
```
Comment (dubeejw) [inline README.md]: "For example, if a post is made with a message of..."

File: README.md [hunk 4]
```diff
@@ -1,4 +1,95 @@
...
+                "amount": 5,
+                "isAwesome": true,
+                "title": "Some string"
+            }
+        }
+    ]
+}
+```
+
+However, if I post the same message content with `isJSONData` set to `false`, my trigger payload would look like this:
```
Comment (dubeejw) [inline README.md]: "However, if a post is made with the same message content..."

File: README.md [hunk 5]
```diff
@@ -1,4 +1,97 @@
...
 ## Status
-This is still very much a work-in-progress. So far a feed provider service and feed action have been implemented. These allow you to create a trigger that fires when new messages are posted to the Kafka instance/topic provided. While this technically works, the current state is still "alpha" quality, and should only be used for kicking the tires.
+This project is currently "beta". Most of the major work is done, and now we are putting on the final touches to get this ready for production use.
+
+## Usage
+This package allows you to create triggers that react when messages are posted to either an IBM Message Hub instance, or to a generic kafka instance. Since the parameters required for each of these situations are different, there are two separate feeds to handle them: `/messaging/messageHubFeed` and `messaging/kafkaFeed`
+
+
+### Creating a Trigger that Listens to an IBM MessageHub Instance
+In order to create a trigger that reacts when messages are posted to a Message Hub instance, you need to use the feed named `messaging/messageHubFeed`. This feed supports the following parameters
```
Comment (dubeejw) [inline README.md]: "This feed supports the following parameters:"

File: README.md [hunk 6]
```diff
@@ -1,4 +1,97 @@
 This project is an OpenWhisk package that allows you to communicate with Kafka or IBM Message Hub instances.
 
 ## Status
-This is still very much a work-in-progress. So far a feed provider service and feed action have been implemented. These allow you to create a trigger that fires when new messages are posted to the Kafka instance/topic provided. While this technically works, the current state is still "alpha" quality, and should only be used for kicking the tires.
+This project is currently "beta". Most of the major work is done, and now we are putting on the final touches to get this ready for production use.
+
+## Usage
+This package allows you to create triggers that react when messages are posted to either an IBM Message Hub instance, or to a generic kafka instance. Since the parameters required for each of these situations are different, there are two separate feeds to handle them: `/messaging/messageHubFeed` and `messaging/kafkaFeed`
```
Comment (dubeejw) [inline README.md]: Add a period at the end of your last sentence.

File: README.md [hunk 7]
```diff
@@ -1,4 +1,97 @@
...
+
+|Name|Type|Description|
+|---|---|---|
+|brokers|JSON Array of Strings|This parameter is an array of `<host>:<port>` strings which comprise the brokers in your Message Hub instance|
+|topic|String|The topic you would like the trigger to listen to|
+|isJSONData|Boolean (Optional - default=false)|When set to `true` this will cause the feed to attempt the message content as JSON before passing it along as the trigger payload.|
+
+Example:
+```
+wsk trigger create MyKafkaTrigger -f /whisk.system/messaging/kafkaFeed -p brokers "[\"mykafkahost:9092\", \"mykafkahost:9093\"]" -p topic mytopic -p isJSONData true
```
Comment (dubeejw) [inline README.md]: Give an example of using the messageHubFeed as well.

File: README.md (hunk 4 above)
Comment (bjustin-ibm) [inline README.md]: "by zombies"

http://auwritingcenter.blogspot.com/2012/10/identify-passive-voice-with-zombies.html

😏

File: README.md (hunk 7 above)
Comment (bjustin-ibm) [inline README.md]: grrr... fine!

**Pull request merged**